packages/web-components/docs/coding-conventions.md

```diff
@@ -437,15 +439,40 @@ To cause re-rendering upon change in `private`/`protected` properties, use `stat
 
 Global `document`/`window` can be different from the ones associated with custom element instance, when the custom element is transported to a different frame e.g. with `document.importNode()`. Though such cases are rare, the codebase avoids global `document`/`window` reference to keep ourselves in a good DOM citizen. We use [`element.ownerDocument`](https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument)/[`.element.ownerDocument.defaultView`](https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView), respectively, instead.
 
-## Custom element registration
+## Custom Elements
+### @carbonElement Decorator
+
+We use a custom `@carbonElement` decorator instead of Lit's `@customElement`, which doesn't provide
```

Comment:
Might be good to link to both our decorator as well as Lit's

Reply:
Good call! I made the update